README.md

```diff
@@ -116,6 +116,39 @@ $ node -e "console.log('Hello from Node.js ' + process.jsEngine)"
 Hello from Node.js chakracore
 ```
 
+#### Mac OS
+
+Prerequisites:
+* XCode v8.2.1
+* Homebrew
+
+Install the building tools:
+
+```
+xcode-select --install
+brew install cmake icu4c
+```
+
+Clone this repo and check out **`xplat`** branch:
+
+```
+git checkout xplat
+```
+
+Build:
+
+```
+./configure --engine=chakracore --dest-os=mac --with-icu-path=/usr/local/opt/icu4c/include --without-intl
```

Comment:
yes, I was experimenting. I will try to re build the Chakra core without the flag.